test/Python/src/HelloWorld.lf

```diff
@@ -2,6 +2,9 @@ target Python {
     tracing: true
 };
 reactor HelloWorld2 {
+    preamble {=
+        import sys
```

Comment:
I'm a bit scratching my head too. 
It seems like the old version of this test contains a syntax error because it is using `self.sys` but it is not importing `sys` inside the namespace of the `HelloWorld2` reactor class.

To give a bit of a context, the following reactor definition should have no syntax errors:
```
reactor HelloWorld2 {
    reaction(startup) {=
        sys.stderr.write("ERROR: startup reaction executed.\n")
    =}
}
```
That is because the `sys` module is already imported globally in the generated code.

But, you can also import `sys` inside a reactor class like it is being done here, by putting it in a preamble in the reactor definition. In that case, the generated code will look like:
```Python
class _HelloWorld2:
    import sys
```
This puts the imported `sys` module within the namespace of the reactor class, and subsequently it can be addressed using the `self.sys` qualifier within the class. 

In the case of this test, the `self` qualifier for `sys` is unnecessary.

Reply:
Soroush is right -- I needed to pass syntax checks, but the way I did it was confusing. It is clearer with the import and the `self` qualifier deleted.

Reply:
It's very cool that we have syntax checks now for Python code. Previously, syntax errors would sometimes only surface when the actual line that contains the syntax error is reached during execution.